lib/autocomplete-manager.coffee

```diff
@@ -393,6 +399,11 @@ class AutocompleteManager
         # Activate on removal of a space, a non-whitespace character, or a bracket-matcher pair
         shouldActivate = oldText is ' ' or oldText.trim().length is 1 or oldText in @bracketMatcherPairs
 
+      # Suppress activation if the editorView has classes that match the suppression list
+      if shouldActivate and @editorView?.classList? and @suppressForClasses?.length
```

Comment:
Well we don't want to do work if someone has `[]` for @suppressForClasses...

I will remove some of the null checking.